packages/web/src/components/features/pages/map/BaseSelector/index.tsx

```
@@ -1,10 +1,11 @@
 import MAP from "@/constants/map";
 import ROUTE from "@/constants/route";
```

ESLint

[error] 'ROUTE' is defined but never used. Allowed unused vars must match /^_/u.

(no-unused-vars)